src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -7728,6 +7718,12 @@ public struct ChunkEnumerator
         void System.Runtime.Serialization.ISerializable.GetObjectData(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) { }
         public override string ToString() { throw null; }
         public string ToString(int startIndex, int length) { throw null; }
+        public partial struct ChunkEnumerator
```

Comment:
From https://github.com/dotnet/corefx/pull/29877

cc @vancem, please review for correctness